python/pg8000-example.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import pg8000.native
+import ssl
+
+ssl_context = ssl.create_default_context()
+ssl_context.check_hostname = ssl.CERT_REQUIRED
```

Comment:
as @vmarcos mentioned in MaterializeInc/materialize#15888, `ssl.CERT_REQUIRED` seems to be the default setting with `ssl.create_default_context()` according to [the `ssl` module docs](https://docs.python.org/3/library/ssl.html#ssl.CERT_REQUIRED). I changed my code as follows to reflect that:

```python
# verify_mode=ssl.CERT_REQUIRED is the default
ssl_context = ssl.create_default_context()
```

Reply:
That is a good point! Just updated this accordingly